.github/workflows/release.yml

```
@@ -8,13 +8,18 @@ jobs:
     uses: ./.github/workflows/ci-tests.yml
     secrets: inherit
 
+  ci-integration-tests:
+    uses: ./.github/workflows/ci-integration-tests.yml
+    needs: ci-tests
+    secrets: inherit
+
   release-build:
     uses: ./.github/workflows/gorelease.yml
-    needs: ci-tests
+    needs: [ci-tests, ci-integration-tests]
```

Small nit, I'm not a fan of json arrays inside a yaml
```suggestion
    needs:
      - ci-tests
      - ci-integration-tests
```